mm2src/coins/eth/erc20.rs

```diff
@@ -90,16 +90,19 @@ pub fn get_erc20_ticker_by_contract_address(ctx: &MmArc, platform: &str, contrac
     })
 }
 
-/// Finds an enabled ERC20 token by its contract address and returns it as `MmCoinEnum`.
-pub async fn get_enabled_erc20_by_contract(
+/// Finds an enabled ERC20 token by contract address and platform coin ticker and returns it as `MmCoinEnum`.
+pub async fn get_enabled_erc20_by_contract_and_platform(
     ctx: &MmArc,
     contract_address: Address,
+    platform: &str,
```

Comment:
super nitty nit: we can order these two parameters so that `platform` is first, making the signautre similar to the function above it (which performs more or less a similar logic).

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2445/commits/4866575879e1e2fa76002ee195690996c0512a2a
reordered params. also noticed unnecessary Address copying, so did small optimisation by using reference